Tests/SentryTests/SentryCrash/SentryCrashJSONCodec_Tests.m

```diff
@@ -1672,4 +1672,20 @@ - (void)testDontCloseLastContainer
     return result;
 }
 
+- (void)testFastUIntEncode
```

Comment:
`m`: Thanks for adding the test, but I think it's not working properly in CI.

Reply:
fixed